build.sbt

```diff
@@ -34,7 +34,7 @@ lazy val commonSettings = Seq(
       // Publish snapshots to LocationTech
       Some("LocationTech Snapshot Repository" at s"${locationtech}/geotrellis-snapshots")
     } else {
-      val milestoenRx = """-M\d+""".r
+      val milestoenRx = """-(M|RC)\d+$""".r
```

Comment:
(nit pick) Is that supposed to be `milestoneRx`?

Reply:
heh, yes, thx